src/GitCommand.jl

```diff
@@ -5,13 +5,11 @@ import ReplMaker
 
 # export @git_cmd
 export git
-export gitrepl
 
 const GIT_REPL_MODE_NAME = "GitCommand.jl Git REPL mode"
 const GIT_REPL_MODE_PROMPT_TEXT = "git> "
 const GIT_REPL_MODE_START_KEY = ','
 
```

Comment:
Probably also these lines should be removed?
```suggestion
```

Reply:
Good catch! Done.